s3fs/utils.py

```diff
@@ -9,6 +10,15 @@ def ignoring(*exceptions):
         pass
 
 
+class FileExpired(IOError):
+    def __init__(self, filename: str, e_tag: str):
+        super().__init__(
+            errno.EBUSY,
```

Comment:
Maybe EBADFD?

Reply:
Yes this was exactly the error code I chose first. However at python 3.8 (windows) it does not exist inside the errno. If you know a better one, which is available all different platforms, please suggest one.

Reply:
I did not know that! I don't think it matters too much, if the message is clear.